include/security.php

```diff
@@ -146,7 +169,7 @@ function authenticate_success($user_record, $login_initial = false, $interactive
 
 		// Set the login date for all identities of the user
 		dba::update('user', ['login_date' => DateTimeFormat::utcNow()],
-			['password' => $master_record['password'], 'email' => $master_record['email'], 'account_removed' => false]);
+			['parent-uid' => $master_record['uid'], 'account_removed' => false]);
```

Comment:
This only works if one logs in as the parent account. If logged with a managed accound, no rows will be updated.

Reply:
Yes. But I don't see it as a problem.

Reply:
Ok then.